packages/syntax-core/src/RichSelect/RichSelectList.tsx

```diff
@@ -0,0 +1,240 @@
+import React, {
+  type ReactElement,
+  useMemo,
+  type SyntheticEvent,
+  useRef,
+} from "react";
+import classNames from "classnames";
+import {
+  ColorBaseDestructive700,
+  ColorBaseGray800,
+} from "@cambly/syntax-design-tokens";
+import Typography from "../Typography/Typography";
+import useIsHydrated from "../useIsHydrated";
+import Popover from "../Popover/Popover";
+import {
+  Label as ReactAriaLabel,
+  Provider as ReactAriaProvider,
+} from "react-aria-components";
+import { useControlledState } from "@react-stately/utils";
+import { DialogContext } from "../Dialog/Dialog";
+import styles from "../SelectList/SelectList.module.css";
+import RichSelectBox, {
+  RichSelectBoxContext,
+  convertSelection,
+  type RichSelectBoxProps,
+} from "./RichSelectBox";
+import richSelectItems from "./richSelectItems";
+import TapArea from "../TapArea/TapArea";
+import { type OverlayHandlerRef } from "../react-aria-utils/Triggerable";
+import Box from "../Box/Box";
+
+const NOOP = () => undefined;
+
+const iconSize = {
+  sm: 20,
+  md: 24,
+  lg: 24,
+} as const;
+
+export type RichSelectListProps = Omit<
+  RichSelectBoxProps,
+  "accessibilityLabel"
+> & {
+  /** Test id for the select element */
+  "data-testid"?: string;
+  /**
+   * Disables the select dropdown entirely
+   * @defaultValue false
+   */
+  disabled?: boolean;
+  /** Callback to be called when select is clicked */
+  onClick?: (event: SyntheticEvent<HTMLDivElement>) => void;
+  /** Text shown below select box if there is an input error. */
+  errorText?: string;
+  /** Text shown below select box */
+  helperText?: string;
+  /** Text shown above select box */
+  label: string; // also show this inside the popover?
+  /**
+   * Text showing in select box if no option has been chosen.
+   * There should always have a placeholder unless there is a default option selected
+   */
+  placeholderText?: string;
+  /** Use to render (override) text shown in the select box */
+  selectTextValue?: (selectedValues?: string[]) => string | undefined;
+  /**
+   * Size of the select box
+   *
+   * @defaultValue "md"
+   */
+  size?: "sm" | "md" | "lg";
+};
+
+/**
+ * [RichSelectList](https://cambly-syntax.vercel.app/?path=/docs/components-selectlist--docs) is a dropdown menu that allows users to select one option from a list.
+ */
+function RichSelectList(props: RichSelectListProps): ReactElement {
+  const {
+    autosave,
+    children,
+    "data-testid": dataTestId,
+    disabled: disabledProp = false,
+    errorText,
+    helperText,
+    label = "myLabel",
+    multiple = false,
+    onChange,
+    onClick = NOOP,
+    placeholderText,
+    primaryButtonText = "Save",
```

Comment:
@benjroy blocking - remove the default values for the `primaryButtonText` / `primaryButtonAccessibilityLabel` / `secondaryButtonText` / `secondaryButtonAccessibilityLabel`

Reply:
@benjroy do the same thing for `RichSelectBox`

Reply:
removed the default values in `RichSelectBox`, and passing `{...richSelectBoxProps}` in `RichSelectList` here: https://github.com/Cambly/syntax/pull/307/commits/9fd2dca8076a4ebc7371f1b4516806b54be26f4c